sql/src/main/java/org/apache/druid/sql/calcite/schema/MetadataSegmentView.java

```diff
@@ -106,10 +106,15 @@ public void start()
     }
     try {
       if (isCacheEnabled) {
-        scheduledExec.schedule(new PollTask(), 0, TimeUnit.MILLISECONDS);
-        lifecycleLock.started();
-        log.info("MetadataSegmentView Started.");
+        try {
+          poll();
+        }
+        finally {
+          scheduledExec.schedule(new PollTask(), 0, TimeUnit.MILLISECONDS);
```

Comment:
Please set the proper wait time instead of starting poll() immediately.

Reply:
yeah, changed to use `pollPeriodinMS`